### ISSUE TITLE
Failure on moc vm

### ISSUE THREAD

**Issue by shuwens** [knikolla/ansible-haas#5]
I created a new  instance and tested the ansible-haas. Looks like I was blocked on the install haas.

I used centos7/ubuntu14.04.

```
[archie@arch: ansible-haas (master)]$ ansible-playbook site.yml -v --limit @site.retry
Using /etc/ansible/ansible.cfg as config file

PLAY [haas] ********************************************************************

TASK [setup] *******************************************************************
ok: [hilserver]

TASK [haas : disable selinux] **************************************************
changed: [hilserver] => {"changed": true, "configfile": "/etc/selinux/config", "msg": "state change will take effect next reboot", "policy": "targeted", "state": "disabled"}

TASK [haas : Install epel-release] *********************************************
ok: [hilserver] => {"changed": false, "msg": "", "rc": 0, "results": []}

TASK [haas : update apt-get cache] *********************************************
skipping: [hilserver] => {"changed": false, "skip_reason": "Conditional check failed", "skipped": true}

TASK [haas : Install dependencies] *********************************************
ok: [hilserver] => (item=[u'libvirt', u'bridge-utils', u'ipmitool', u'telnet', u'python-pip', u'python-lxml', u'python-devel', u'qemu-kvm', u'virt-install', u'gcc', u'git', u'wget']) => {"changed": false, "item": ["libvirt", "bridge-utils", "ipmitool", "telnet", "python-pip", "python-lxml", "python-devel", "qemu-kvm", "virt-install", "gcc", "git", "wget"], "msg": "", "rc": 0, "results": ["libvirt-1.2.17-13.el7_2.5.x86_64 providing libvirt is already installed", "bridge-utils-1.5-9.el7.x86_64 providing bridge-utils is already installed", "ipmitool-1.8.13-8.el7_1.x86_64 providing ipmitool is already installed", "telnet-1:0.17-59.el7.x86_64 providing telnet is already installed", "python-pip-7.1.0-1.el7.noarch providing python-pip is already installed", "python-lxml-3.2.1-4.el7.x86_64 providing python-lxml is already installed", "python-devel-2.7.5-34.el7.x86_64 providing python-devel is already installed", "qemu-kvm-10:1.5.3-105.el7_2.4.x86_64 providing qemu-kvm is already installed", "virt-install-1.2.1-8.el7.noarch providing virt-install is already installed", "gcc-4.8.5-4.el7.x86_64 providing gcc is already installed", "git-1.8.3.1-6.el7_2.1.x86_64 providing git is already installed", "wget-1.14-10.el7_0.1.x86_64 providing wget is already installed"]}

TASK [haas : create haas user] *************************************************
ok: [hilserver] => {"append": false, "changed": false, "comment": "HaaS user", "group": 1001, "home": "/var/lib/haas", "move_home": false, "name": "haas", "shell": "/bin/bash", "state": "present", "uid": 1001}

TASK [haas : git clone HaaS] ***************************************************
ok: [hilserver] => {"after": "3e361f3d6de3131bae32dcc190027ad5dadb0ca6", "before": "3e361f3d6de3131bae32dcc190027ad5dadb0ca6", "changed": false, "warnings": []}

TASK [haas : install haas] *****************************************************
fatal: [hilserver]: FAILED! => {"changed": true, "cmd": "cd /var/lib/haas/haas; python setup.py install", "delta": "0:00:07.358132", "end": "2016-07-08 03:14:10.980285", "failed": true, "rc": 1, "start": "2016-07-08 03:14:03.622153", "stderr": "In file included from src/lxml/lxml.etree.c:320:0:\nsrc/lxml/includes/etree_defs.h:14:31: fatal error: libxml/xmlversion.h: No such file or directory\n #include \"libxml/xmlversion.h\"\n                               ^\ncompilation terminated.\n/tmp/easy_install-h6RLUK/lxml-3.6.0/temp/xmlXPathInit0UvqTM.c:1:26: fatal error: libxml/xpath.h: No such file or directory\n #include \"libxml/xpath.h\"\n                          ^\ncompilation terminated.\nerror: Setup script exited with error: command 'gcc' failed with exit status 1", "stdout": "running install\nrunning bdist_egg\nrunning egg_info\nwriting requirements to haas.egg-info/requires.txt\nwriting haas.egg-info/PKG-INFO\nwriting top-level names to haas.egg-info/top_level.txt\nwriting dependency_links to haas.egg-info/dependency_links.txt\nwriting entry points to haas.egg-info/entry_points.txt\nreading manifest file 'haas.egg-info/SOURCES.txt'\nreading manifest template 'MANIFEST.in'\nwriting manifest file 'haas.egg-info/SOURCES.txt'\ninstalling library code to build/bdist.linux-x86_64/egg\nrunning install_lib\nrunning build_py\ncreating build/bdist.linux-x86_64/egg\ncreating build/bdist.linux-x86_64/egg/haas\ncopying build/lib/haas/__init__.py -> build/bdist.linux-x86_64/egg/haas\ncopying build/lib/haas/api.py -> build/bdist.linux-x86_64/egg/haas\ncopying build/lib/haas/auth.py -> build/bdist.linux-x86_64/egg/haas\ncopying build/lib/haas/class_resolver.py -> build/bdist.linux-x86_64/egg/haas\ncopying build/lib/haas/cli.py -> build/bdist.linux-x86_64/egg/haas\ncopying build/lib/haas/config.py -> build/bdist.linux-x86_64/egg/haas\ncopying build/lib/haas/deferred.py -> build/bdist.linux-x86_64/egg/haas\ncopying build/lib/haas/dev_support.py -> build/bdist.linux-x86_64/egg/haas\ncopying build/lib/haas/errors.py -> build/bdist.linux-x86_64/egg/haas\ncopying build/lib/haas/flaskapp.py -> build/bdist.linux-x86_64/egg/haas\ncopying build/lib/haas/migrations.py -> build/bdist.linux-x86_64/egg/haas\ncopying build/lib/haas/model.py -> build/bdist.linux-x86_64/egg/haas\ncopying build/lib/haas/network_allocator.py -> build/bdist.linux-x86_64/egg/haas\ncopying build/lib/haas/rest.py -> build/bdist.linux-x86_64/egg/haas\ncopying build/lib/haas/server.py -> build/bdist.linux-x86_64/egg/haas\ncopying build/lib/haas/test_common.py -> build/bdist.linux-x86_64/egg/haas\ncreating build/bdist.linux-x86_64/egg/haas/commands\ncopying build/lib/haas/commands/__init__.py -> build/bdist.linux-x86_64/egg/haas/commands\ncopying build/lib/haas/commands/admin.py -> build/bdist.linux-x86_64/egg/haas/commands\ncopying build/lib/haas/commands/db.py -> build/bdist.linux-x86_64/egg/haas/commands\ncreating build/bdist.linux-x86_64/egg/haas/ext\ncopying build/lib/haas/ext/__init__.py -> build/bdist.linux-x86_64/egg/haas/ext\ncreating build/bdist.linux-x86_64/egg/haas/ext/auth\ncopying build/lib/haas/ext/auth/__init__.py -> build/bdist.linux-x86_64/egg/haas/ext/auth\ncopying build/lib/haas/ext/auth/database.py -> build/bdist.linux-x86_64/egg/haas/ext/auth\ncopying build/lib/haas/ext/auth/mock.py -> build/bdist.linux-x86_64/egg/haas/ext/auth\ncopying build/lib/haas/ext/auth/null.py -> build/bdist.linux-x86_64/egg/haas/ext/auth\ncreating build/bdist.linux-x86_64/egg/haas/ext/network_allocators\ncopying build/lib/haas/ext/network_allocators/__init__.py -> build/bdist.linux-x86_64/egg/haas/ext/network_allocators\ncopying build/lib/haas/ext/network_allocators/null.py -> build/bdist.linux-x86_64/egg/haas/ext/network_allocators\ncopying build/lib/haas/ext/network_allocators/vlan_pool.py -> build/bdist.linux-x86_64/egg/haas/ext/network_allocators\ncreating build/bdist.linux-x86_64/egg/haas/ext/obm\ncopying build/lib/haas/ext/obm/__init__.py -> build/bdist.linux-x86_64/egg/haas/ext/obm\ncopying build/lib/haas/ext/obm/ipmi.py -> build/bdist.linux-x86_64/egg/haas/ext/obm\ncopying build/lib/haas/ext/obm/mock.py -> build/bdist.linux-x86_64/egg/haas/ext/obm\ncreating build/bdist.linux-x86_64/egg/haas/ext/obm/migrations\ncreating build/bdist.linux-x86_64/egg/haas/ext/obm/migrations/mock\ncopying build/lib/haas/ext/obm/migrations/mock/df8d9f423f2b_rename_mockobm_table_for_flask.py -> build/bdist.linux-x86_64/egg/haas/ext/obm/migrations/mock\ncreating build/bdist.linux-x86_64/egg/haas/ext/switches\ncopying build/lib/haas/ext/switches/__init__.py -> build/bdist.linux-x86_64/egg/haas/ext/switches\ncopying build/lib/haas/ext/switches/_console.py -> build/bdist.linux-x86_64/egg/haas/ext/switches\ncopying build/lib/haas/ext/switches/brocade.py -> build/bdist.linux-x86_64/egg/haas/ext/switches\ncopying build/lib/haas/ext/switches/dell.py -> build/bdist.linux-x86_64/egg/haas/ext/switches\ncopying build/lib/haas/ext/switches/mock.py -> build/bdist.linux-x86_64/egg/haas/ext/switches\ncopying build/lib/haas/ext/switches/nexus.py -> build/bdist.linux-x86_64/egg/haas/ext/switches\ncopying build/lib/haas/ext/switches/null.py -> build/bdist.linux-x86_64/egg/haas/ext/switches\ncreating build/bdist.linux-x86_64/egg/haas/ext/switches/migrations\ncreating build/bdist.linux-x86_64/egg/haas/ext/switches/migrations/brocade\ncopying build/lib/haas/ext/switches/migrations/brocade/5a6db7a7222d_added_brocade_driver.py -> build/bdist.linux-x86_64/egg/haas/ext/switches/migrations/brocade\ncreating build/bdist.linux-x86_64/egg/haas/ext/switches/migrations/dell\ncopying build/lib/haas/ext/switches/migrations/dell/099b939261c1_rename_dell_switch_table_for_flask_.py -> build/bdist.linux-x86_64/egg/haas/ext/switches/migrations/dell\ncreating build/bdist.linux-x86_64/egg/haas/ext/switches/migrations/mock\ncopying build/lib/haas/ext/switches/migrations/mock/b5b31d19257d_rename_mockswitch_table_for_flask_.py -> build/bdist.linux-x86_64/egg/haas/ext/switches/migrations/mock\ncreating build/bdist.linux-x86_64/egg/haas/migrations\ncopying build/lib/haas/migrations/env.py -> build/bdist.linux-x86_64/egg/haas/migrations\ncopying build/lib/haas/migrations/alembic.ini -> build/bdist.linux-x86_64/egg/haas/migrations\ncopying build/lib/haas/migrations/script.py.mako -> build/bdist.linux-x86_64/egg/haas/migrations\ncreating build/bdist.linux-x86_64/egg/haas/migrations/versions\ncopying build/lib/haas/migrations/versions/6a8c19565060_move_to_flask.py -> build/bdist.linux-x86_64/egg/haas/migrations/versions\nbyte-compiling build/bdist.linux-x86_64/egg/haas/__init__.py to __init__.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/api.py to api.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/auth.py to auth.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/class_resolver.py to class_resolver.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/cli.py to cli.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/config.py to config.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/deferred.py to deferred.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/dev_support.py to dev_support.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/errors.py to errors.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/flaskapp.py to flaskapp.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/migrations.py to migrations.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/model.py to model.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/network_allocator.py to network_allocator.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/rest.py to rest.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/server.py to server.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/test_common.py to test_common.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/commands/__init__.py to __init__.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/commands/admin.py to admin.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/commands/db.py to db.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/__init__.py to __init__.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/auth/__init__.py to __init__.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/auth/database.py to database.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/auth/mock.py to mock.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/auth/null.py to null.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/network_allocators/__init__.py to __init__.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/network_allocators/null.py to null.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/network_allocators/vlan_pool.py to vlan_pool.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/obm/__init__.py to __init__.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/obm/ipmi.py to ipmi.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/obm/mock.py to mock.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/obm/migrations/mock/df8d9f423f2b_rename_mockobm_table_for_flask.py to df8d9f423f2b_rename_mockobm_table_for_flask.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/switches/__init__.py to __init__.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/switches/_console.py to _console.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/switches/brocade.py to brocade.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/switches/dell.py to dell.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/switches/mock.py to mock.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/switches/nexus.py to nexus.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/switches/null.py to null.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/switches/migrations/brocade/5a6db7a7222d_added_brocade_driver.py to 5a6db7a7222d_added_brocade_driver.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/switches/migrations/dell/099b939261c1_rename_dell_switch_table_for_flask_.py to 099b939261c1_rename_dell_switch_table_for_flask_.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/ext/switches/migrations/mock/b5b31d19257d_rename_mockswitch_table_for_flask_.py to b5b31d19257d_rename_mockswitch_table_for_flask_.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/migrations/env.py to env.pyc\nbyte-compiling build/bdist.linux-x86_64/egg/haas/migrations/versions/6a8c19565060_move_to_flask.py to 6a8c19565060_move_to_flask.pyc\ncreating build/bdist.linux-x86_64/egg/EGG-INFO\ninstalling scripts to build/bdist.linux-x86_64/egg/EGG-INFO/scripts\nrunning install_scripts\nrunning build_scripts\ncreating build/bdist.linux-x86_64/egg/EGG-INFO/scripts\ncopying build/scripts-2.7/haas -> build/bdist.linux-x86_64/egg/EGG-INFO/scripts\ncopying build/scripts-2.7/create_bridges -> build/bdist.linux-x86_64/egg/EGG-INFO/scripts\nchanging mode of build/bdist.linux-x86_64/egg/EGG-INFO/scripts/haas to 755\nchanging mode of build/bdist.linux-x86_64/egg/EGG-INFO/scripts/create_bridges to 755\ncopying haas.egg-info/PKG-INFO -> build/bdist.linux-x86_64/egg/EGG-INFO\ncopying haas.egg-info/SOURCES.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\ncopying haas.egg-info/dependency_links.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\ncopying haas.egg-info/entry_points.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\ncopying haas.egg-info/not-zip-safe -> build/bdist.linux-x86_64/egg/EGG-INFO\ncopying haas.egg-info/requires.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\ncopying haas.egg-info/top_level.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\ncreating 'dist/haas-0.2rc2-py2.7.egg' and adding 'build/bdist.linux-x86_64/egg' to it\nremoving 'build/bdist.linux-x86_64/egg' (and everything under it)\nProcessing haas-0.2rc2-py2.7.egg\nremoving '/usr/lib/python2.7/site-packages/haas-0.2rc2-py2.7.egg' (and everything under it)\ncreating /usr/lib/python2.7/site-packages/haas-0.2rc2-py2.7.egg\nExtracting haas-0.2rc2-py2.7.egg to /usr/lib/python2.7/site-packages\nhaas 0.2rc2 is already the active version in easy-install.pth\nInstalling haas script to /usr/bin\nInstalling create_bridges script to /usr/bin\nInstalling haas-admin script to /usr/bin\n\nInstalled /usr/lib/python2.7/site-packages/haas-0.2rc2-py2.7.egg\nProcessing dependencies for haas==0.2rc2\nSearching for lxml>=3.6.0,<4.0\nReading https://pypi.python.org/simple/lxml/\nBest match: lxml 3.6.0\nDownloading https://pypi.python.org/packages/11/1b/fe6904151b37a0d6da6e60c13583945f8ce3eae8ebd0ec763ce546358947/lxml-3.6.0.tar.gz#md5=5957cc384bd6e83934be35c057ec03b6\nProcessing lxml-3.6.0.tar.gz\nWriting /tmp/easy_install-h6RLUK/lxml-3.6.0/setup.cfg\nRunning lxml-3.6.0/setup.py -q bdist_egg --dist-dir /tmp/easy_install-h6RLUK/lxml-3.6.0/egg-dist-tmp-mSQUGh\nBuilding lxml version 3.6.0.\nBuilding without Cython.\nERROR: /bin/sh: xslt-config: command not found\n\n** make sure the development packages of libxml2 and libxslt are installed **\n\nUsing build configuration of libxslt \nCompile failed: command 'gcc' failed with exit status 1\n*********************************************************************************\nCould not find function xmlCheckVersion in library libxml2. Is libxml2 installed?\n*********************************************************************************", "stdout_lines": ["running install", "running bdist_egg", "running egg_info", "writing requirements to haas.egg-info/requires.txt", "writing haas.egg-info/PKG-INFO", "writing top-level names to haas.egg-info/top_level.txt", "writing dependency_links to haas.egg-info/dependency_links.txt", "writing entry points to haas.egg-info/entry_points.txt", "reading manifest file 'haas.egg-info/SOURCES.txt'", "reading manifest template 'MANIFEST.in'", "writing manifest file 'haas.egg-info/SOURCES.txt'", "installing library code to build/bdist.linux-x86_64/egg", "running install_lib", "running build_py", "creating build/bdist.linux-x86_64/egg", "creating build/bdist.linux-x86_64/egg/haas", "copying build/lib/haas/__init__.py -> build/bdist.linux-x86_64/egg/haas", "copying build/lib/haas/api.py -> build/bdist.linux-x86_64/egg/haas", "copying build/lib/haas/auth.py -> build/bdist.linux-x86_64/egg/haas", "copying build/lib/haas/class_resolver.py -> build/bdist.linux-x86_64/egg/haas", "copying build/lib/haas/cli.py -> build/bdist.linux-x86_64/egg/haas", "copying build/lib/haas/config.py -> build/bdist.linux-x86_64/egg/haas", "copying build/lib/haas/deferred.py -> build/bdist.linux-x86_64/egg/haas", "copying build/lib/haas/dev_support.py -> build/bdist.linux-x86_64/egg/haas", "copying build/lib/haas/errors.py -> build/bdist.linux-x86_64/egg/haas", "copying build/lib/haas/flaskapp.py -> build/bdist.linux-x86_64/egg/haas", "copying build/lib/haas/migrations.py -> build/bdist.linux-x86_64/egg/haas", "copying build/lib/haas/model.py -> build/bdist.linux-x86_64/egg/haas", "copying build/lib/haas/network_allocator.py -> build/bdist.linux-x86_64/egg/haas", "copying build/lib/haas/rest.py -> build/bdist.linux-x86_64/egg/haas", "copying build/lib/haas/server.py -> build/bdist.linux-x86_64/egg/haas", "copying build/lib/haas/test_common.py -> build/bdist.linux-x86_64/egg/haas", "creating build/bdist.linux-x86_64/egg/haas/commands", "copying build/lib/haas/commands/__init__.py -> build/bdist.linux-x86_64/egg/haas/commands", "copying build/lib/haas/commands/admin.py -> build/bdist.linux-x86_64/egg/haas/commands", "copying build/lib/haas/commands/db.py -> build/bdist.linux-x86_64/egg/haas/commands", "creating build/bdist.linux-x86_64/egg/haas/ext", "copying build/lib/haas/ext/__init__.py -> build/bdist.linux-x86_64/egg/haas/ext", "creating build/bdist.linux-x86_64/egg/haas/ext/auth", "copying build/lib/haas/ext/auth/__init__.py -> build/bdist.linux-x86_64/egg/haas/ext/auth", "copying build/lib/haas/ext/auth/database.py -> build/bdist.linux-x86_64/egg/haas/ext/auth", "copying build/lib/haas/ext/auth/mock.py -> build/bdist.linux-x86_64/egg/haas/ext/auth", "copying build/lib/haas/ext/auth/null.py -> build/bdist.linux-x86_64/egg/haas/ext/auth", "creating build/bdist.linux-x86_64/egg/haas/ext/network_allocators", "copying build/lib/haas/ext/network_allocators/__init__.py -> build/bdist.linux-x86_64/egg/haas/ext/network_allocators", "copying build/lib/haas/ext/network_allocators/null.py -> build/bdist.linux-x86_64/egg/haas/ext/network_allocators", "copying build/lib/haas/ext/network_allocators/vlan_pool.py -> build/bdist.linux-x86_64/egg/haas/ext/network_allocators", "creating build/bdist.linux-x86_64/egg/haas/ext/obm", "copying build/lib/haas/ext/obm/__init__.py -> build/bdist.linux-x86_64/egg/haas/ext/obm", "copying build/lib/haas/ext/obm/ipmi.py -> build/bdist.linux-x86_64/egg/haas/ext/obm", "copying build/lib/haas/ext/obm/mock.py -> build/bdist.linux-x86_64/egg/haas/ext/obm", "creating build/bdist.linux-x86_64/egg/haas/ext/obm/migrations", "creating build/bdist.linux-x86_64/egg/haas/ext/obm/migrations/mock", "copying build/lib/haas/ext/obm/migrations/mock/df8d9f423f2b_rename_mockobm_table_for_flask.py -> build/bdist.linux-x86_64/egg/haas/ext/obm/migrations/mock", "creating build/bdist.linux-x86_64/egg/haas/ext/switches", "copying build/lib/haas/ext/switches/__init__.py -> build/bdist.linux-x86_64/egg/haas/ext/switches", "copying build/lib/haas/ext/switches/_console.py -> build/bdist.linux-x86_64/egg/haas/ext/switches", "copying build/lib/haas/ext/switches/brocade.py -> build/bdist.linux-x86_64/egg/haas/ext/switches", "copying build/lib/haas/ext/switches/dell.py -> build/bdist.linux-x86_64/egg/haas/ext/switches", "copying build/lib/haas/ext/switches/mock.py -> build/bdist.linux-x86_64/egg/haas/ext/switches", "copying build/lib/haas/ext/switches/nexus.py -> build/bdist.linux-x86_64/egg/haas/ext/switches", "copying build/lib/haas/ext/switches/null.py -> build/bdist.linux-x86_64/egg/haas/ext/switches", "creating build/bdist.linux-x86_64/egg/haas/ext/switches/migrations", "creating build/bdist.linux-x86_64/egg/haas/ext/switches/migrations/brocade", "copying build/lib/haas/ext/switches/migrations/brocade/5a6db7a7222d_added_brocade_driver.py -> build/bdist.linux-x86_64/egg/haas/ext/switches/migrations/brocade", "creating build/bdist.linux-x86_64/egg/haas/ext/switches/migrations/dell", "copying build/lib/haas/ext/switches/migrations/dell/099b939261c1_rename_dell_switch_table_for_flask_.py -> build/bdist.linux-x86_64/egg/haas/ext/switches/migrations/dell", "creating build/bdist.linux-x86_64/egg/haas/ext/switches/migrations/mock", "copying build/lib/haas/ext/switches/migrations/mock/b5b31d19257d_rename_mockswitch_table_for_flask_.py -> build/bdist.linux-x86_64/egg/haas/ext/switches/migrations/mock", "creating build/bdist.linux-x86_64/egg/haas/migrations", "copying build/lib/haas/migrations/env.py -> build/bdist.linux-x86_64/egg/haas/migrations", "copying build/lib/haas/migrations/alembic.ini -> build/bdist.linux-x86_64/egg/haas/migrations", "copying build/lib/haas/migrations/script.py.mako -> build/bdist.linux-x86_64/egg/haas/migrations", "creating build/bdist.linux-x86_64/egg/haas/migrations/versions", "copying build/lib/haas/migrations/versions/6a8c19565060_move_to_flask.py -> build/bdist.linux-x86_64/egg/haas/migrations/versions", "byte-compiling build/bdist.linux-x86_64/egg/haas/__init__.py to __init__.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/api.py to api.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/auth.py to auth.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/class_resolver.py to class_resolver.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/cli.py to cli.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/config.py to config.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/deferred.py to deferred.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/dev_support.py to dev_support.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/errors.py to errors.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/flaskapp.py to flaskapp.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/migrations.py to migrations.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/model.py to model.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/network_allocator.py to network_allocator.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/rest.py to rest.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/server.py to server.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/test_common.py to test_common.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/commands/__init__.py to __init__.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/commands/admin.py to admin.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/commands/db.py to db.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/__init__.py to __init__.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/auth/__init__.py to __init__.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/auth/database.py to database.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/auth/mock.py to mock.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/auth/null.py to null.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/network_allocators/__init__.py to __init__.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/network_allocators/null.py to null.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/network_allocators/vlan_pool.py to vlan_pool.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/obm/__init__.py to __init__.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/obm/ipmi.py to ipmi.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/obm/mock.py to mock.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/obm/migrations/mock/df8d9f423f2b_rename_mockobm_table_for_flask.py to df8d9f423f2b_rename_mockobm_table_for_flask.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/switches/__init__.py to __init__.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/switches/_console.py to _console.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/switches/brocade.py to brocade.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/switches/dell.py to dell.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/switches/mock.py to mock.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/switches/nexus.py to nexus.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/switches/null.py to null.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/switches/migrations/brocade/5a6db7a7222d_added_brocade_driver.py to 5a6db7a7222d_added_brocade_driver.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/switches/migrations/dell/099b939261c1_rename_dell_switch_table_for_flask_.py to 099b939261c1_rename_dell_switch_table_for_flask_.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/ext/switches/migrations/mock/b5b31d19257d_rename_mockswitch_table_for_flask_.py to b5b31d19257d_rename_mockswitch_table_for_flask_.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/migrations/env.py to env.pyc", "byte-compiling build/bdist.linux-x86_64/egg/haas/migrations/versions/6a8c19565060_move_to_flask.py to 6a8c19565060_move_to_flask.pyc", "creating build/bdist.linux-x86_64/egg/EGG-INFO", "installing scripts to build/bdist.linux-x86_64/egg/EGG-INFO/scripts", "running install_scripts", "running build_scripts", "creating build/bdist.linux-x86_64/egg/EGG-INFO/scripts", "copying build/scripts-2.7/haas -> build/bdist.linux-x86_64/egg/EGG-INFO/scripts", "copying build/scripts-2.7/create_bridges -> build/bdist.linux-x86_64/egg/EGG-INFO/scripts", "changing mode of build/bdist.linux-x86_64/egg/EGG-INFO/scripts/haas to 755", "changing mode of build/bdist.linux-x86_64/egg/EGG-INFO/scripts/create_bridges to 755", "copying haas.egg-info/PKG-INFO -> build/bdist.linux-x86_64/egg/EGG-INFO", "copying haas.egg-info/SOURCES.txt -> build/bdist.linux-x86_64/egg/EGG-INFO", "copying haas.egg-info/dependency_links.txt -> build/bdist.linux-x86_64/egg/EGG-INFO", "copying haas.egg-info/entry_points.txt -> build/bdist.linux-x86_64/egg/EGG-INFO", "copying haas.egg-info/not-zip-safe -> build/bdist.linux-x86_64/egg/EGG-INFO", "copying haas.egg-info/requires.txt -> build/bdist.linux-x86_64/egg/EGG-INFO", "copying haas.egg-info/top_level.txt -> build/bdist.linux-x86_64/egg/EGG-INFO", "creating 'dist/haas-0.2rc2-py2.7.egg' and adding 'build/bdist.linux-x86_64/egg' to it", "removing 'build/bdist.linux-x86_64/egg' (and everything under it)", "Processing haas-0.2rc2-py2.7.egg", "removing '/usr/lib/python2.7/site-packages/haas-0.2rc2-py2.7.egg' (and everything under it)", "creating /usr/lib/python2.7/site-packages/haas-0.2rc2-py2.7.egg", "Extracting haas-0.2rc2-py2.7.egg to /usr/lib/python2.7/site-packages", "haas 0.2rc2 is already the active version in easy-install.pth", "Installing haas script to /usr/bin", "Installing create_bridges script to /usr/bin", "Installing haas-admin script to /usr/bin", "", "Installed /usr/lib/python2.7/site-packages/haas-0.2rc2-py2.7.egg", "Processing dependencies for haas==0.2rc2", "Searching for lxml>=3.6.0,<4.0", "Reading https://pypi.python.org/simple/lxml/", "Best match: lxml 3.6.0", "Downloading https://pypi.python.org/packages/11/1b/fe6904151b37a0d6da6e60c13583945f8ce3eae8ebd0ec763ce546358947/lxml-3.6.0.tar.gz#md5=5957cc384bd6e83934be35c057ec03b6", "Processing lxml-3.6.0.tar.gz", "Writing /tmp/easy_install-h6RLUK/lxml-3.6.0/setup.cfg", "Running lxml-3.6.0/setup.py -q bdist_egg --dist-dir /tmp/easy_install-h6RLUK/lxml-3.6.0/egg-dist-tmp-mSQUGh", "Building lxml version 3.6.0.", "Building without Cython.", "ERROR: /bin/sh: xslt-config: command not found", "", "** make sure the development packages of libxml2 and libxslt are installed **", "", "Using build configuration of libxslt ", "Compile failed: command 'gcc' failed with exit status 1", "*********************************************************************************", "Could not find function xmlCheckVersion in library libxml2. Is libxml2 installed?", "*********************************************************************************"], "warnings": []}

NO MORE HOSTS LEFT *************************************************************
        to retry, use: --limit @site.retry

PLAY RECAP *********************************************************************
hilserver                  : ok=6    changed=1    unreachable=0    failed=1   
```

**Comment by knikolla** [comment]
@shwsun Sorry, just saw this. Will fix. The VM is missing a dependency for building lxml from source.
